solidity/contracts/converter/ConverterVersion.sol

```diff
@@ -2,5 +2,5 @@
 pragma solidity 0.6.12;
 
 contract ConverterVersion {
-    uint16 public constant version = 45;
+    uint16 public constant version = 46;
```

Comment:
This will "collide" with the same change in my PR, or even worse - it will not "yell" when I merge my (identical) change.
Assuming that we want an increment of at most 1 on every release **to master**, perhaps we should both revert this change, and do it only before releasing the current version (0.6.34).